include/odp/api/spec/traffic_mngr.h

```diff
@@ -351,6 +355,26 @@ typedef struct {
 	 * only as many number of WFQ groups as MAX priority.
 	 */
 	uint8_t max_wfq_groups_per_node;
+
+	/** tm_queue_threshold_supported indicates support for threshold
+	 * profile on a TM queue. When TRUE, users can set/clear/update
+	 * threshold profile on a TM queue. When false clear none of it is
+	 * supported.
+	 */
+	odp_bool_t tm_queue_threshold_supported;
+
+	/** tm_queue_query_mask indicates supported types of TM queue query.
+	 * Types of TM queue query are same as query_flags that are passed to
+	 * odp_tm_queue_query(), odp_tm_priority_query() and
+	 * odp_tm_total_query(). When zero, none of the queue query API's are
+	 * supported. When non-zero, only the supported fields are filled
+	 * irrespective of the query_flags passed and rest of the fields are
+	 * untouched.
+	 *
+	 * @see ODP_TM_QUERY_PKT_CNT, ODP_TM_QUERY_BYTE_CNT,
+	 * ODP_TM_QUERY_THRESHOLDS.
+	 */
+	uint8_t tm_queue_query_mask;
```

Comment:
Since query flags in the query functions is uint32_t, this should be too.

Reply:
Ack.